Doc/library/functions.rst

```diff
@@ -1262,7 +1262,8 @@ are always available.  They are listed here in alphabetical order.
    closest multiple of 10 to the power minus *ndigits*; if two multiples are
```

Comment:
@haypo: agree:

> Return *number* rounded to *ndigits* digits after the decimal point.

seems good to me.

About the negation, placement is good (too soon would have diluted the definition). About its wording, I'd prefer telling what it does instead of just stating it's possible, something like:

> Negative *ndigits* permit rouding before the decimal point.